tests/test_working_copy_sqlserver.py

```diff
@@ -585,52 +593,137 @@ def test_checkout_custom_crs(
                 '-     UNIT["degree", 0.0174532925199433,',
                 '-         AUTHORITY["EPSG", "9122"]],',
                 '-     AUTHORITY["EPSG", "4326"]]',
-                "- ",
-                "--- nz_pa_points_topo_150k:meta:schema.json",
-                "+++ nz_pa_points_topo_150k:meta:schema.json",
-                "  [",
-                "    {",
+                '- ',
```

Comment:
pre-commit hook not running? Ruff should be converting these to double-quotes

`pre-commit install`

Reply:
Re-committed everything from my mac where this is working, haven't got around to fixing it on my VM